code/bart_stability.R

```diff
@@ -54,6 +54,35 @@ get_data <- function(dgp,n, q, rho=0.05, seed=0){
 
 }
 
+# Assumes that the dgp argument is in c("2016","2017","2018","2019")
+# Throws an error if the n + 200 > the number of rows in the data
+get_data_acic <- function(dgp,n){
+
+  train_n <- n
+  test_n <- 200
+
+  if (!(dgp %in% c("2016","2017","2018","2019"))) {
+    stop("Not a supported dgp")
+  }
+
+  data <- read.csv(paste0("data/ACIC",dgp,".csv"))
+
+  if (nrow(data) < train_n+test_n) {
+    stop("N is too large for the current dgp")
+  }
+
+  data_train <- data[1:train_n,-which(colnames(data) == "Ytrue")]
+  data_test <- data[(train_n+1):(train_n+200),-which(colnames(data) == "Y")]
+
+  colnames(data_train)[which(colnames(data_train) == "Y")] <- "y"
```

Comment:
The columns names create a bug with bart_fit, could you please make sure the columns names are the same for train and test? Also could you please check that the CLI works using acic arguments?